src/pages/DetailsPage.js

```diff
@@ -101,7 +101,15 @@ const DetailsPage = ({personalDetails, route}) => {
                                     <Text style={[styles.textP]} numberOfLines={1}>
                                         {moment().tz(details.timezone.selected).format('LT')}
                                         {' '}
-                                        {moment().tz(details.timezone.selected).zoneAbbr()}
+                                        {isNaN(moment().tz(details.timezone.selected).zoneAbbr()) ? ( 
```

Comment:
Would you mind moving the logic here up above the return? So basically make a `currentTime` var and then here it will just be `<Text>{timezone}</Text>`